### DISCUSSION TITLE
ROG Zephyrus M16 GU603ZX

### DISCUSSION THREAD

**Discussion by SpoonerUK** [seerge/g-helper#197] [Q&A] [answered]
Just wondering if anyone has got this working on a 2022 GU603ZX ?  ( i9 12900H / RTX3080TI )

I do hope so, and am hoping for a confirmation before I go ahead and throw ArmouryCrate out of the window.

I am getting repeated micro-stutter on this thing, and I am almost certain that it's AC doing it.
Micro-Stutter in this case refers to my system just freezing up very frequently for one or two seconds at a time.  Usually when browsing in Chrome / Edge, or just using Windows (File) Explorer.  When gaming, I don't seem to get it.

**Comment by seerge** [comment, marked as answer]
@SpoonerUK - people use it, there is a whole post on reddit about it https://www.reddit.com/r/ZephyrusM16/comments/11xgdqa/installed_ghelper_on_i93070ti_m16/

You can just try it , it's not necessary to uninstall armoury to check if app works or not

**Comment by SpoonerUK** [comment]
Thanks a lot for the response Serge.

Seems to be fine for me.

1) Disabled control interface in BIOS.
2) Ran the debloat.bat (after checking what it does.)
3) Used System Restore to create a restore point.
4) Uninstalled ArmouryCrate using the Asus tool.
5) Rebooted, set ghelper.exe to launch on startup.
6) Profit.

<img width="308" alt="image" src="https://user-images.githubusercontent.com/87868144/230303923-f5e8177e-b1b9-42cb-ae85-c2ae25aab52a.png">
